src/test/regress/expected/shard_move_deferred_delete.out

```diff
@@ -196,7 +196,7 @@ $cmd$);
  run_command_on_workers
 ---------------------------------------------------------------------
  (localhost,57637,t,1)
- (localhost,57638,t,1)
```

Comment:
why did this change?

Reply:
because that orphaned shard now gets cleaned up